packages/terra-horizontal-divider/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Horizontal Divider
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-horizontal-divider.svg)](https://www.npmjs.org/package/terra-horizontal-divider)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+{insert description}
```

Comment:
Add description

Reply:
I added in the next commit https://github.com/cerner/terra-core/pull/962/commits/2644419f5511b5699f07e99aad57480c42b8d8b4#diff-3fe4bec3374f1238c4e258d3c3e199feR5

Reply:
You added it in the /docs/README you're still missing it in this file.

Reply:
Sorry my bad, updated https://github.com/cerner/terra-core/pull/962/commits/9f8a5deba40296bf91341b6ff1d0d56f4bae16ac#diff-04320eabdc4b26475f2ecf331caec423R4

Reply:
My bad, updated https://github.com/cerner/terra-core/commit/5fb27563217e211a5ec708f6dc65da66fb369f31#diff-04320eabdc4b26475f2ecf331caec423L6